README.md

```diff
@@ -14,10 +14,32 @@ pipeline, you will have to replace the `DATADOG_AGENT_BUILDIMAGES` variable
 in the [.gitlab-ci.yml](https://github.com/DataDog/datadog-agent/blob/master/.gitlab-ci.yml)
 of the [datadog-agent repository][agent] to use the newly created images.
 
+## If your changes requires a mirror PR
+
+Current tests are running a [datadog-agent pipeline][agent] without kitchen 
+tests to confirm if your newly builded images are working. If your changes requires
+a mirror PR on the [datadog-agent repository][agent], you have two option :
+### If you need multiple test commits in agent-buildimages
+  - Add `branch: your/datadog-agent-branch` in your [.gitlab-ci.yml](https://github.com/DataDog/datadog-agent-buildimages/blob/fcc4843103d3bfdb976da845133ad3edc48754b2/.gitlab-ci.yml#L261-L263)
+  - Commit and waits for the `dd-gitlab/wait_for_tests` jobs to appear in the CI, it will indicate if your tests succeeded or failed.
+    Once your tests worked you can move to the next step.
+  - Remove `branch: your/datadog-agent-branch` in your [.gitlab-ci.yml](https://github.com/DataDog/datadog-agent-buildimages/blob/fcc4843103d3bfdb976da845133ad3edc48754b2/.gitlab-ci.yml#L261-L263)
+    - Commit and don't waits for the `dd-gitlab/wait_for_tests` job to appear in the CI. 
+      If your pipeline is green it's good to merge (if you wait too long the `dd-gitlab/wait_for_tests` job will appear failling your PR but this jobs isn't required)
```

Comment:
```suggestion
### If you need multiple test commits in agent-buildimages
In your [datadog-agent][agent]'s PR:
  - Add `branch: your/datadog-agent-branch` in your [.gitlab-ci.yml](https://github.com/DataDog/datadog-agent-buildimages/blob/fcc4843103d3bfdb976da845133ad3edc48754b2/.gitlab-ci.yml#L261-L263) file.
  - Commit and wait for the `dd-gitlab/wait_for_tests` job to appear in the CI, it will indicate if your tests succeeded or failed. Once your tests worked you can move on to the next step.
  - Remove `branch: your/datadog-agent-branch` in your [.gitlab-ci.yml](https://github.com/DataDog/datadog-agent-buildimages/blob/fcc4843103d3bfdb976da845133ad3edc48754b2/.gitlab-ci.yml#L261-L263) file.
  - Commit and don't wait for the `dd-gitlab/wait_for_tests` job to appear in the CI. If your pipeline is green it's good to merge (if you wait for too long the `dd-gitlab/wait_for_tests` job will appear failing your PR but this jobs isn't required)
```

Reply:
Here it's actually into the agent-buildimages PR not datadog-agent

Reply:
I'm going to put `datadog-agent-buildimages's PR` instead